projects/Amlogic/filesystem/usr/lib/udev/hwdb.d/70-local-keyboard.hwdb

```diff
@@ -0,0 +1,2 @@
+evdev:input:b0005v0217p0000e0110*
```

Comment:
this should be PR'd separately and in a generic way.

Reply:
OK, this will be dropped.